lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -42,6 +53,28 @@ def get_object(self, queryset=None):
 
         raise Http404
 
+    def update(self, request, *args, **kwargs):
```

Comment:
Use a mixin for this. An example is provided at http://www.django-rest-framework.org/api-guide/generic-views/#put-as-create.